Added initial examples (#672)
ROCmSoftwarePlatform/AMDMIGraphX: examples/migraphx_driver/README.md

# MIGraphX Driver

> There is a new driver as /opt/rocm/bin/migraphx-driver.  The --help command shows options available

Reviewer: ```suggestion
The MIGraphX driver is located under /opt/rocm/bin/migraphx-driver.  The --help command shows options available
```

```
prompt$ /opt/rocm/bin/migraphx-driver --help

    -h, --help
        Show help

Commands:
    perf
    params
    read
    run
    verify
    compile
```
For example, the read command will read a file and print the internal graph from MIGraphx
```

> /opt/rocm/bin/migraphx-driver read --onnx /home/mev/source/migraphx_onnx/torchvision/resnet50i64.onnx 

Reviewer: can you add a small snippet (maybe up to 5 lines max) of the output?

```
Another example, the following command measures performance running an ONNX file using the driver
```

> /opt/rocm/bin/migraphx-driver perf --onnx /home/mev/source/migraphx_onnx/torchvision/resnet50i64.onnx 

Reviewer: can you include the performance summary part of the output?

```

> The verify command checks internal consistency once read into MIGraphX

Reviewer: ```suggestion
The verify command is a correctness checker and runs both the reference and GPU implementation to ensure the model outputs are consistent 
```

```
/opt/rocm/bin/migraphx-driver verify --onnx /home/mev/source/migraphx_onnx/torchvision/resnet50i64.onnx 

> ```

Reviewer: can you include a brief summary of the expected output after running (i.e., no errors emitted if verify passes)?

# MIGraphX Driver

## Description
The MIGraphX driver is a tool that allows you to utilize many of the core functions of MIGraphX without having to write your own program. 

## How to Use this Example

> The MIGraphX driver is built alongside MIGraphX and can be found in `/opt/rocm/bin/migraphx-driver`

Reviewer 2: Can you add "or in the folder "build/bin" after building the source code?"

Author: Added
